core/src/main/java/org/mule/runtime/core/internal/streaming/CursorManager.java

```diff
@@ -21,7 +22,9 @@
 import com.github.benmanes.caffeine.cache.LoadingCache;
 
 import java.lang.ref.WeakReference;
+import java.util.Collections;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.Collections'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7688&issues=AWj9Gu0AotIXnmsUljPh&open=AWj9Gu0AotIXnmsUljPh)

[ISSUE_KEY]: AWj9Gu0AotIXnmsUljPh